packages/frontend/src/store/timeEntries.ts

```diff
@@ -79,30 +79,26 @@ const actions = {
 
       commit("FLUSH_PUSH_QUEUE");
       try {
-        const response = await adAuthenticatedFetch(
-          config.API_HOST + "/api/user/TimeEntries",
-          {
-            method: "post",
-            headers: {
-              "Content-Type": "application/json",
-            },
-            body: JSON.stringify(timeEntriesToPush),
-          }
-        );
-        const timeEntries = await response.json();
-        if (response.status !== 200) {
-          throw Error(`${response.statusText}
-${timeEntries.title}`);
-        }
-        if (!Array.isArray(timeEntries) && timeEntries.message) {
-          throw Error(timeEntries.message);
-        }
-        if (Array.isArray(timeEntries)) {
-          commit(
-            "UPDATE_TIME_ENTRIES_AFTER_UPDATE",
-            timeEntries.map(createTimeEntrie)
-          );
-        }
+        await httpClient
+          .post<any[]>(
```

Comment:
Kan denne types til en type? 

Hvis det er vanskelig går det også an å type det til input-et til `createTimeEntrie`;

```ts
Array<Parameters<typeof createTimeEntrie>[0]>
```

Men det er kanskje ikke det klareste eller mest leselige i verden

Reply:
Jeg ser nå at parameterene til `createTimeEntrie` er `any` også. Kanskje vi kan type det vi vet vi trenger ved å endre på `createTimeEntrie`? 

```ts
function createTimeEntrie<T extends { date: string, value: number }>(data: T): FrontendTimentrie {
```

og 

```suggestion
          .post<Array<Parameters<typeof createTimeEntrie>[0]>>(
```

Det vil i hvert fall være en start på å få solid typing